### ISSUE TITLE
External catalog tables not seen in Database Navigator when using Redshift Spectrum.

### ISSUE THREAD

**Issue by BigDataDaddy** [serge-rider/dbeaver#1897]
Redshift recently added support for querying external tables in AWS S3 as described by an external "Hive like" catalog that is serviced either by the AWS Athena Data Catalog Service (based on the Hive Metastore Service) or an actual Hive Metastore Service, like on an AWS EMR cluster.   

When declaring an external schema in Redshift, the schema itself shows up as empty, but the external tables in the schema are not showing up in the "Database Navigator" in DBeaver.  

I suspect DBeaver does not yet know how to query the contents of the Redshift external catalog to see the external tables.

**Comment by serge-rider** [comment]
Yes.
I'll need to setup S3 Athena to test/fix this.
Thanks for report.

**Comment by BigDataDaddy** [comment]
This was fixed by the AWS Redshift JDBC driver release  1.2.7.1003 downloadable here: http://docs.aws.amazon.com/redshift/latest/mgmt/configure-jdbc-connection.html.  I've tested and am using the new driver in DBeaver EE v4.0.5. 
 
The Redshift JDBC driver without the AWS SDK classes injected into the driver is the one you what to choose to use with DBeaver due to classloader conflicts.  I'm not using it, but I think you'll need to add the AWS SDK jar(s) to your classpath in the DBeaver Driver Manager in order to use IAM mediated authentication and SSO features introduced in another feature of this driver version.
